.eslintrc

```diff
@@ -1,7 +1,3 @@
 {
-  "extends": "./.config/.eslintrc",
-  "plugins": ["prettier"],
```

Comment:
Do you not want to have prettier lints anymore?

Reply:
We added this back then to see prettier errors during development time and also catch format errors. See https://raintank-corp.slack.com/archives/C01C4K8DETW/p1695739670894709

Reply:
>This is the recommended way from create plugin scaffolding.

We're only passing on the recommendation from prettier authors. ([Docs link](https://prettier.io/docs/integrating-with-linters#notes))

We added `prettier --write` to `lint:fix` command in create-plugin@5.12.0. The change is in this [PR package.json changes](https://github.com/grafana/grafana-infinity-datasource/pull/1242/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R43). If you would like to catch format errors I would suggest adding git commit hooks to lint / format files on commit.